templates/parachain/runtime/src/genesis_config_presets.rs

```diff
@@ -17,14 +24,19 @@ use sp_keyring::Sr25519Keyring;
 /// The default XCM version to set in genesis config.
 const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
 /// Parachain id used for genesis config presets of parachain template.
-pub const PARACHAIN_ID: u32 = 1000;
+#[export]
+pub const PARACHAIN_ID: u32 = 2000;
 
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
 pub fn template_session_keys(keys: AuraId) -> SessionKeys {
 	SessionKeys { aura: keys }
 }
+#[docify::export(name = "get_parachain_id")]
+pub fn get_parachain_id() -> u32 {
+	PARACHAIN_ID
+}
```

Comment:
Thinking again about this, I would have this function return a plain `1000` (such that it corresponds with the cost `PARACHAIN_ID`). We should follow with a test that asserts `PARACHAIN_ID == get_parachain_id()`. Might sound redundant, but this is to overcome the lack of docify support around interpolating exported primitive constants by value directly into the README.

Then, we can directly use the `get_parachain_id` tag in the README.docify.md to interpolate 1000 into the doc wherever it is needed. I haven't tried this though, I have a slight concern it might not work everywhere.

Last thing would be to run `cargo build --features generate-readme` to trigger the README.md compilation, based on the README.docify.md file.

```suggestion
const fn get_parachain_id() -> u32 {
	1000
}
```